src/Symfony/Component/Messenger/Bridge/AmazonSqs/Tests/Transport/AmazonSqsTransportTest.php

```diff
@@ -177,6 +177,9 @@ private function getTransport(SerializerInterface $serializer = null, Connection
 
     private function createHttpException(): HttpException
     {
-        return new ServerException($this->createMock(ResponseInterface::class));
+        $response = $this->createMock(ResponseInterface::class);
+        $response->method('getInfo')->with('http_code')->willReturn(500);
```

Comment:
To fix all the tests on the Messenger component:

```suggestion
        $response->method('getInfo')->with('http_code')->willReturn(500);
        $response->method('getInfo')->with('url')->willReturn('https://symfony.com');
```